vue.config.js

```diff
@@ -1,6 +1,22 @@
 module.exports = {
-  publicPath: process.env.NODE_ENV === 'production' ? '/vue-typescript-admin-template/' : '/',
+  publicPath: '/hengwei/security/',
+
   pwa: {
     name: 'vue-typescript-admin-template'
+  },
+
+  configureWebpack: {
```

Comment:
Better make sure each of below config is useful or need to be set in the project.